### PR TITLE
Update orchestration docs to use ellipses syntax

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Update orchestration docs to use ellipses syntax

## Motivation and Context
Orchestration docs cleanup

## How to test this PR?
Just test the docker-compose changes locally using `docker-compose` command

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [x] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#f95c2b38-c3ea-4852-9feb-40a674856d20) |
| mint-large-bucket.sh | :x: [more...](#fcb650aa-1d6f-4143-b3f0-46c36c8e6463) |
<a name="fcb650aa-1d6f-4143-b3f0-46c36c8e6463"></a>
### 8153-68d6480/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:30812
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5c5ad089be79:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 56 seconds
(4/14) Running aws-sdk-ruby tests ... done in 16 seconds
(5/14) Running awscli tests ... done in 2 minutes and 35 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 15 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 31 seconds
(9/14) Running minio-go tests ... done in 2 minutes and 23 seconds
(10/14) Running minio-java tests ... FAILED in 2 minutes and 49 seconds
{
  "name": "minio-java",
  "function": "presignedGetObject(String bucketName, String objectName)",
  "duration": 20108,
  "status": "FAIL",
  "error": "java.net.SocketTimeoutException: timeout >>> [okio.Okio$4.newTimeoutException(Okio.java:232), okio.AsyncTimeout.exit(AsyncTimeout.java:286), okio.AsyncTimeout$2.read(AsyncTimeout.java:241), okio.RealBufferedSource.indexOf(RealBufferedSource.java:358), okio.RealBufferedSource.readUtf8LineStrict(RealBufferedSource.java:230), okhttp3.internal.http1.Http1Codec.readHeaderLine(Http1Codec.java:226), okhttp3.internal.http1.Http1Codec.readResponseHeaders(Http1Codec.java:202), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:101), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:264), okhttp3.RealCall.execute(RealCall.java:93), FunctionalTest.readObject(FunctionalTest.java:183), FunctionalTest.presignedGetObject_test1(FunctionalTest.java:1767), FunctionalTest.runTests(FunctionalTest.java:2994), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="f95c2b38-c3ea-4852-9feb-40a674856d20"></a>
### 8153-68d6480/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32439
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d52a6400f2a3:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 53 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 43 seconds
(8/14) Running minio-dotnet tests ... done in 28 seconds
(9/14) Running minio-go tests ... done in 47 seconds
(10/14) Running minio-java tests ... FAILED in 25 seconds
{
  "name": "minio-java",
  "function": "removeIncompleteUpload(String bucketName, String objectName)",
  "duration": 1994,
  "status": "FAIL",
  "error": "java.net.SocketException: Connection reset >>> [java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:115), java.net.SocketOutputStream.write(SocketOutputStream.java:155), okio.Okio$1.write(Okio.java:79), okio.AsyncTimeout$1.write(AsyncTimeout.java:180), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.write(RealBufferedSink.java:42), okhttp3.internal.http1.Http1Codec$FixedLengthSink.write(Http1Codec.java:306), okio.ForwardingSink.write(ForwardingSink.java:35), okhttp3.internal.http.CallServerInterceptor$CountingSink.write(CallServerInterceptor.java:163), okio.RealBufferedSink.emitCompleteSegments(RealBufferedSink.java:179), okio.RealBufferedSink.writeAll(RealBufferedSink.java:107), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:74), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:83), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:264), okhttp3.RealCall.execute(RealCall.java:93), io.minio.MinioClient.executeReq(MinioClient.java:1111), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executePut(MinioClient.java:1405), io.minio.MinioClient.executePut(MinioClient.java:1427), io.minio.MinioClient.putObject(MinioClient.java:4216), io.minio.MinioClient.putObject(MinioClient.java:4328), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.removeIncompleteUploads_test(FunctionalTest.java:1719), FunctionalTest.runTests(FunctionalTest.java:2992), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```

File: docs/orchestration/kubernetes/minio-distributed-statefulset.yaml [hunk 1]
```diff
@@ -24,10 +24,7 @@ spec:
         image: minio/minio:RELEASE.2019-08-29T00-25-01Z
         args:
         - server
-        - http://minio-0.minio.default.svc.cluster.local/data
-        - http://minio-1.minio.default.svc.cluster.local/data
-        - http://minio-2.minio.default.svc.cluster.local/data
-        - http://minio-3.minio.default.svc.cluster.local/data
+        - http://minio-{0...1}.minio.default.svc.cluster.local/data
```
Comment (kannappanr) [inline docs/orchestration/kubernetes/minio-distributed-statefulset.yaml]:
```suggestion
        - http://minio-{0...3}.minio.default.svc.cluster.local/data
```

**Review by kannappanr — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#0818ee5f-ad19-4698-b5cf-8476a6b28c81) |
<a name="0818ee5f-ad19-4698-b5cf-8476a6b28c81"></a>
### 8153-0509a56/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:31296
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0a35c99f2363:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 2 minutes and 17 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 1 minutes and 13 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 14 seconds
(9/14) Running minio-go tests ... FAILED in 4 minutes and 5 seconds
{
  "args": {},
  "duration": 99294,
  "error": "The specified key does not exist.",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "Stat failed",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```

File: docs/orchestration/kubernetes/minio-distributed-statefulset.yaml (hunk 1 above)
Comment (nitisht) [reply docs/orchestration/kubernetes/minio-distributed-statefulset.yaml]: This should be `minio-{0...3}`, looks like didn't get updated

**Review by nitisht — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#fec0a092-7d6f-42c0-b688-d583f4b05709) |
| mint-large-bucket.sh | :x: [more...](#a53e0d76-7ea2-464d-845c-39b3b0535273) |
<a name="a53e0d76-7ea2-464d-845c-39b3b0535273"></a>
### 8153-c0f02ca/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:30919
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 33e879e756c0:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 21 seconds
(5/14) Running awscli tests ... done in 1 minutes and 35 seconds
(6/14) Running healthcheck tests ... done in 2 seconds
(7/14) Running mc tests ... done in 1 minutes and 34 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 27 seconds
(9/14) Running minio-go tests ... FAILED in 3 minutes and 41 seconds
{
  "args": {},
  "duration": 79984,
  "error": "The specified key does not exist.",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "Stat failed",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```
<a name="fec0a092-7d6f-42c0-b688-d583f4b05709"></a>
### 8153-c0f02ca/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:30050
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 1442e77689d5:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-p010gydn42jjzu",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360015,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15C05BD01EF888E3, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#af0013dc-d4c2-4a41-a647-017a473bf43d) |
<a name="af0013dc-d4c2-4a41-a647-017a473bf43d"></a>
### 8153-e55178a/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30607
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp cce046e7312e:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... FAILED in 6 minutes and 3 seconds
{
  "name": "aws-sdk-php",
  "function": "getBucketPolicy ( array $params = [] )",
  "args": {
    "Bucket": "aws-sdk-php-5d6bf37991be8"
  },
  "duration": "40857",
  "status": "FAIL",
  "error": "Error executing \"PutObject\" on \"http://72.28.97.53:30607/aws-sdk-php-5d6bf37991be8/test-anon\"; AWS HTTP error: Client error: `PUT http://72.28.97.53:30607/aws-sdk-php-5d6bf37991be8/test-anon` resulted in a `400 Bad Request` response:\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>IncompleteBody</Code><Message>You did not provide the number of byte (truncated...)\n IncompleteBody (client): You did not provide the number of bytes specified by the Content-Length HTTP header. - <?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>IncompleteBody</Code><Message>You did not provide the number of bytes specified by the Content-Length HTTP header.</Message><Key>test-anon</Key><BucketName>aws-sdk-php-5d6bf37991be8</BucketName><Resource>/aws-sdk-php-5d6bf37991be8/test-anon</Resource><RequestId>15C05E193FF65B86</RequestId><HostId>9419571e-01e0-4e63-a39f-bfa3f9d34077</HostId></Error>"
}

Executed 2 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#a212ba0c-5bba-4c53-9595-d701255744a3) |
<a name="a212ba0c-5bba-4c53-9595-d701255744a3"></a>
### 8153-25a698b/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30992
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a87c84f65b53:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 16 seconds
(5/14) Running awscli tests ... done in 2 minutes and 19 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 1 minutes and 27 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 33 seconds
(9/14) Running minio-go tests ... done in 2 minutes and 39 seconds
(10/14) Running minio-java tests ... done in 3 minutes and 5 seconds
(11/14) Running minio-js tests ... done in 1 minutes and 5 seconds
(12/14) Running minio-py tests ... FAILED in 5 minutes and 45 seconds
{
  "function": "list_objects(bucket_name, prefix, recursive)",
  "message": "BucketNotEmpty: message: The bucket you tried to delete is not empty.",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 1168, in test_list_objects_with_1001_files\n    client.remove_bucket(bucket_name)\n  File \"/usr/local/lib/python3.5/dist-packages/minio/api.py\", line 417, in remove_bucket\n    self._url_open('DELETE', bucket_name=bucket_name)\n  File \"/usr/local/lib/python3.5/dist-packages/minio/api.py\", line 1952, in _url_open\n    object_name).get_exception()\nminio.error.BucketNotEmpty: BucketNotEmpty: message: The bucket you tried to delete is not empty.\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 2024, in main\n    test_list_objects_with_1001_files(client, log_output)\n  File \"/mint/run/core/minio-py/tests.py\", line 1170, in test_list_objects_with_1001_files\n    raise Exception(err)\nException: BucketNotEmpty: message: The bucket you tried to delete is not empty.\n",
  "name": "minio-py:test_list_objects_with_1001_files",
  "args": {
    "object_name": "d26ff860-6c19-4378-9a55-f9b42b24ec6b_0 ~ d26ff860-6c19-4378-9a55-f9b42b24ec6b_1000",
    "bucket_name": "minio-py-test-bf385f4f-775b-48c3-9124-a47bb319379c"
  },
  "status": "FAIL",
  "duration": 293903
}

Executed 11 out of 14 tests successfully.
```

**Pull request merged**